site/en/docs/privacy-sandbox/fedcm-updates/index.md

```diff
@@ -37,6 +37,19 @@ If you're new to this API or have not experimented with it yet, read the
 To stay updated on the FedCM API changes, please subscribe to [our
 newsletter](https://groups.google.com/g/fedcm-developer-newsletter).
 
+### Chrome 117 (Sep 2023)
+
+- Idp Sign-In Status API is now supported on Android Chrome 117 as well. Learn more from [FedCM updates](/blog/fedcm-chrome-116-updates/#idp-signin-status).
+
+### Chrome 116 (Aug 2023)
+
+- Added support for following three new features in Chrome 116
+  - Login Hint API: Specify a preferred user account to be signed in.
+  - User Info API: Fetch the information of the returning user so that the identity provider (IdP) can render a personalized sign-in button within an iframe.
```

Comment:
```suggestion
  - **User Info API**: Fetch the information of the returning user so that the identity provider (IdP) can render a personalized sign-in button within an iframe.
```